website/src/pages/index.tsx

```diff
@@ -106,11 +106,12 @@ function Hero(): JSX.Element {
         <div className="text-center lg:w-2/3 w-full bg-hero-pattern bg-no-repeat bg-center">
           <div className="bg-white/30 dark:bg-transparent">
             <h1 className="title-font sm:text-4xl text-3xl lg:text-6xl mb-8 font-medium text-gray-900 dark:text-white leading-[1.2]">
-              Containers and Kubernetes for application developers
+              Best Free & Open Source Tool for Containers & Kubernetes - Podman Desktop
```

Comment:
I suppose you need to escape these characters
```suggestion
              Best Free &amp; Open Source Tool for Containers &amp; Kubernetes - Podman Desktop
```